style.css

```diff
@@ -0,0 +1,201 @@
+  .wrap {
+    max-width: 1920px;
+    min-width: 375px;
+    height: 100vh;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/VeronikasBaliProject/pullRequest?prid=5927042)